### PR TITLE
gradle settings

### PR DESCRIPTION
**Pull request by dominikl**
Adjust some gradle settings:
- `omero-blitz` dependency needs to be added as 'api'
-  Change the root project name to `omero-java-gateway` too.

@jburel added travis settings.
TODO: use new method ``setCaseSensitive`` when blitz-m5 is tagged

### CODE REVIEW

File: src/main/java/omero/gateway/facility/SearchFacility.java [hunk 1]
```diff
@@ -112,7 +112,7 @@ public SearchResultCollection search(SecurityContext ctx, SearchParameters conte
                 // set general parameters
                 service.clearQueries();
                 service.setAllowLeadingWildcard(true);
-                service.setCaseSensitive(false);
+                service.setCaseSentivice(false);
```
Comment (joshmoore) [inline src/main/java/omero/gateway/facility/SearchFacility.java]: We'll need a reminder to roll this back. Alternatively, we go ahead and release m5 across the board and that is what the first omero-gateway consumes.

Comment (jburel) [reply src/main/java/omero/gateway/facility/SearchFacility.java]: I was going to add a comment to the PR so it gets updated when m5 is out

**Comment by jburel** [comment]
swapping blitz for gateway in insight has not been successful so far.

**Comment by dominikl** [comment]
Good to merge?

File: build.gradle [hunk 2]
```diff
@@ -10,16 +10,13 @@ version = "5.5.0-SNAPSHOT"
...
 dependencies {
-    implementation "org.openmicroscopy:omero-blitz:5.5.0-SNAPSHOT"
-    implementation "commons-beanutils:commons-beanutils:1.9.3"
-    implementation "com.zeroc:icegrid:3.6.4"
-    implementation "commons-lang:commons-lang:2.5"
-    implementation "commons-collections:commons-collections:3.2"
-    testCompile "org.testng:testng:6.9.8"
+    api("org.openmicroscopy:omero-blitz:5.5.0-m4")
+    implementation("commons-beanutils:commons-beanutils:1.9.3")
+    implementation("com.zeroc:icegrid:3.6.4")
```
Comment (joshmoore) [inline build.gradle]: I'm surprised this is necessary. Does `omero-blitz` not expose this? cc: @jburel @rgozim

Comment (jburel) [reply build.gradle]: it should be exposed

Comment (jburel) [reply build.gradle]: Actually they are not.
Without those 2 dependencies it does not build

**Comment by jburel** [comment]
I think that tests can be enabled in a follow-up PR

**Comment by jburel** [comment]
see https://travis-ci.org/ome/omero-java-gateway/builds/514360847

**Comment by joshmoore** [comment]
Running locally with `-i`:

```
model.unit > model.unit.active > omero.gateway.util.UtilsTest.testConvertFrom STANDARD_OUT
    08:51:32.208 [Test worker] DEBUG o.a.c.b.converters.BooleanConverter - Setting default value: false
    08:51:32.216 [Test worker] DEBUG o.a.c.b.converters.BooleanConverter - Converting 'Boolean' value 'false' to type 'Boolean'
    08:51:32.216 [Test worker] DEBUG o.a.c.b.converters.BooleanConverter -     No conversion required, value is already a Boolean
    08:51:32.220 [Test worker] DEBUG o.a.c.b.converters.ByteConverter - Setting default value: 0
    08:51:32.220 [Test worker] DEBUG o.a.c.b.converters.ByteConverter - Converting 'Integer' value '0' to type 'Byte'
    08:51:32.220 [Test worker] DEBUG o.a.c.b.converters.ByteConverter -     Converted to Byte value '0'
    08:51:32.221 [Test worker] DEBUG o.a.c.b.c.CharacterConverter - Setting default value:
...
```

:+1:

**Comment by jburel** [comment]
Merging for the first tag

**Pull request merged**